src/brainseg/create_fh_seg.py

```diff
@@ -0,0 +1,121 @@
+import os
+import sys
+import subprocess
+import nibabel as nib
+import numpy as np
+import scipy.ndimage
+import shutil
+
+def run_synthseg(conda_env, input_nifti, output_nifti, synthseg_script, synthseg_dir):
+    python_bin = os.path.join(conda_env, "bin", "python")
+    env = os.environ.copy()
+    env["PYTHONPATH"] = synthseg_dir
+
+    subprocess.run(
+        [python_bin, synthseg_script, "--i", input_nifti, "--o", output_nifti, "--robust"],
+        check=True, cwd=synthseg_dir, env=env
+    )
+
+def run_charm(conda_env, subject_id, input_nifti, charm_working_dir):
+    charm_bin = os.path.join(conda_env, "bin", "charm")
+    subject_dir = os.path.join(charm_working_dir, f"m2m_{subject_id}")
+
+    os.makedirs(charm_working_dir, exist_ok=True)
+
+    if os.path.exists(subject_dir):
+        shutil.rmtree(subject_dir)
+
+    subprocess.run(
```

Comment:
Similar approach than above for SynthSeg. We need to really activate the conda env